app/helpers/orders_helper.rb

```diff
@@ -23,6 +23,15 @@ def options_for_suppliers_to_select
     options_for_select(options)
   end
 
+  def format_amount(amount, order_article, strip_insignificant_zeros: false)
+    strip_insignificant_zeros = true unless order_article.article_version.supplier_order_unit_is_si_convertible
+    number_with_precision(amount, precision: 3, strip_insignificant_zeros: strip_insignificant_zeros)
+  end
```

Comment:
- This method is solely used inside the helper -> should be private
- Its name isn't very conclusive - format _how_? (Maybe `format_amount_for_export`?)

Reply:
I changed it to be private, so the name is not so important.